pandas/sparse/series.py

```diff
@@ -29,12 +29,14 @@
 
 from pandas.util.decorators import Appender
 
+from pandas.sparse.scipy_sparse import sparse_series_to_coo, coo_to_sparse_series
```

Comment:
I think we can't just import this here, as scipy is only an optional dependency. 
Maybe putting it inside the method is enough? Or does it need a try/except?